virus/virTemplate.md

```diff
@@ -1,40 +1,106 @@
+
+All figures and related information pertaining to dataset runs and Posa can be found in the "MGY441 A3 Virus Seq Runs" csv file.
+[Uploading MGY441 A3 Virus Seq Runs.csv…]()
+[MGY441 A3 Virus Seq Runs.csv](https://github.com/DerekTea/VIRUSxDISCVRY/files/13552458/MGY441.A3.Virus.Seq.Runs.csv)
+
 ### Ecology subsection A...
 
-{Q2 answer ...}
+Facts about Posa-like viruses:
+Posa-like viruses have an RNA genome (Riboviria) [Munnink, 2017]
+- Posa-like viruses have a positive-sense genome [Munnink, 2017]
+- Posa-like viruses have a single-stranded genome [Munnink, 2017]
+- They are non-enveloped RNA viruses [Aoki, 2019]
+- They are small and typically approximately 30 nm in diameter [Yinda, 2017]
+- These viruses are an unclassified family of the Picornavirales order [Munnink, 2017]
+- The length of Picornavirales viruses typically range from 7.2 to 10.2kb [Han, 2020]
+- Posa-like viruses are typically around 2500-3000 amino acids in length [Han, 2020]
+- Viruses within the Picornavirales order are highly diverse, but typically encode a polyprotein composed of a conserved replication block. This replication block contains a helicase, protease, and a RNA-dependent RNA polymerase [Han, 2020]
+- Generally, members of the Picornavirales order are monopartite (all genes are on a single nucleic acid strand) [Munnink, 2017]
+- Posa-like viruses are found in stool samples of various host organisms (posavirus in pigs, hursavirus in humans, rasavirus in rats, fisavirus in fish) [Munnink, 2017]
+- They have been found to infect vetebrates and invertebrates [Han, 2020]
+- Posa-like viruses are generally not associated with pathogenic traits and are typically found in the stool samples of healthy organisms [Munnink, 2017]
+- Posa-like viruses have a broad genetic diversity as amino acid similarity is often less than 40% [Munnink, 2017]
+- There are 3 main genomic organizations within Picornavirales [Han, 2020]
+1) Non-structural module located at the 5' end, structural module located at the 3' end, intergenic region in between
+2) The two modules are located in separate genomic segments 
+3) Non-structural module located at the 3' end, structural module located at the 5' end, intergenic region in between
+- The in Picornavirales genome, the RNA is linked to the viral protein at its 5' end [Jiang, 2014]
+- Picornavirales have a poly-A at its 3' end [Jiang, 2014]
+- The OFR of the Picornavirales genome is flanked by a long 5' non-translated region and a short 3' non-translated region [Jiang, 2014]
+- A prominent conserved RNA secondary structure among Picornavirales includes an internal ribosomal entry site (IRES) towards the 5' end [Witwer, 2015]
+- Picornavirales have a capsid protein with icosahedral symmetry [Jiang, 2014]
 
 ### Virus Genome {Q3}
```

Comment:
Convert this to `.png` and include it embedded as an image. The figure legend should be written into the markdown file given below the figure.